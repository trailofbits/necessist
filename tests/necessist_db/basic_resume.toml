args = ["--root=examples/basic", "--resume"]
stderr = ""

[bin]
name = "necessist"

[status]
code = 0

[env.add]
TRYCMD="1"
