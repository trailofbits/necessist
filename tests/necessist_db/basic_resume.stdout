4 candidates, 0 remaining in 0 test files
